src/redis/encrypt.ts

```diff
@@ -45,15 +45,16 @@ const crc16 = (data: string) => {
   return (~res) & 0x0ffff;
 };
 
-export const hashKey = (key: string): number | null => {
-  if(!key) return null;
+export const getKeyToHash = (key: string): string => {
   const openingBracketIndex = key.indexOf('{');
   const closingBracketIndex = key.indexOf('}', openingBracketIndex);
+  const isHashTag = openingBracketIndex === -1 || closingBracketIndex === -1; 
+  const keyToHash = isHashTag ? key : key.slice(openingBracketIndex + 1, closingBracketIndex);
```

Comment:
A bit confusing because `isHashTag` holds the opposite value - 
if `openingBracketIndex === -1 || closingBracketIndex === -1` is `true` means that the key isn't contains hashtag.

Reply:
@or-ben-dahan renamed to `isPlainKey`